azure-iot-device/azure/iot/device/iothub/aio/async_clients.py

```diff
@@ -350,6 +384,26 @@ async def notify_blob_upload_status(
         await handle_result(callback)
         logger.info("Successfully notified blob upload status")
 
+    @property
+    def on_method_request_received(self):
+        return self._on_method_request_received
+
+    @on_method_request_received.setter
+    def on_method_request_received(self, value):
```

Comment:
>on_method_request_received [](start = 8, length = 26)

similar setter and getters will eventually exist for the message received and twin patches right ?

Reply:
Yes. In fact, this shouldn't even be here yet - this is only going to be added in a separate PR - I will remove before merging.

---
In reply to: [454011418](https://github.com/Azure/azure-iot-sdk-python/pull/620#discussion_r454011418) [](ancestors = 454011418)